File: solutions/1-Two-Sum/twosum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i in range(len(nums)):
            if (target-nums[i]) in nums and nums.index(target-nums[i]) != i:
                return [i, nums.index(target-nums[i])]

bruh = Solution()
print(bruh.twoSum([3,2,4], 6))

File: solutions/1-Two-Sum/twosum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var prevMap = [Int:Int]()
        for (index, value) in nums.enumerated() {
            if let diff = prevMap[value] {
                return [diff, index]
            } else {
                prevMap[target - value] = index
            }
        } 
        return []
    }
}

let bruh: Solution = Solution()
let nums = [3,2,4]
print(bruh.twoSum(nums, 6))